### ISSUE TITLE
How should we handle i18n for FSE templates?

### ISSUE THREAD

**Issue by iandunn** [WordPress/wporg-news-2021#19]
It sounds like blockbase had to create PHP templates

* https://github.com/Automattic/themes/pull/4332
* https://github.com/Automattic/themes/issues/3653

...and [TwentyTwentyTwo had to create patterns](https://github.com/WordPress/twentytwentytwo/issues/30).

those may be good arounds for us too, but still seems pretty hacky. it doesn't sound like gutenberg really supports i18n yet.

* https://github.com/WordPress/gutenberg/issues/27402
* https://github.com/WordPress/gutenberg/issues/21932
* https://github.com/WordPress/gutenberg/issues/36061
* https://github.com/WordPress/gutenberg/issues/21204

will that be an issue when we launch this? i guess we'd continue using the old theme on rosetta sites anyway, since those just use the `main` theme for the entire site, rather than having a separate WP site for the blog.

**Comment by coreymckrill** [comment]
> i guess we'd continue using the old theme on rosetta sites anyway, since those just use the main theme for the entire site, rather than having a separate WP site for the blog.

Ah this is good to know. In that case, I would guess that updating the design on rosetta sites would be a separate project. I would lean towards not worrying about translations for this project right now. There won't be a ton of strings, so it hopefully wouldn't be hard to add in translations once GB supports them.

**Comment by iandunn** [comment]
That sounds good to me 👍🏻

**Comment by iandunn** [comment]
Re-opening since we'll want to do this eventually, but moving it to the farthest-away milestone.

Maybe blocked by #13. We'll probably want to move this to the parent theme repo, and then backport to News.

**Comment by MaggieCabrera** [comment]
We can use block patterns for this. Check how Blockbase does it for the [404 pages](https://github.com/Automattic/themes/blob/41f85140ec63c73fff36f263cf04695a15c7868c/blockbase/block-templates/404.html#L5) while hiding the [block pattern](https://github.com/Automattic/themes/blob/41f85140ec63c73fff36f263cf04695a15c7868c/blockbase/inc/patterns/404.php#L11) from the users in the editor.

**Comment by iandunn** [comment]
Yeah, that could be an acceptable workaround in the meantime, but I'm hoping one of the Gutenberg tickets above will provide something analogous to how PHP/JS files are translated. IMO that'd be a much better solution.

**Comment by MaggieCabrera** [comment]
> Yeah, that could be an acceptable workaround in the meantime, but I'm hoping one of the Gutenberg tickets above will provide something analogous to how PHP/JS files are translated. IMO that'd be a much better solution.

From my experience and what I've talked with other GB contributors I don't think that's happening any time soon and it's not something easy to solve or clear as to how to do so
